### PR TITLE
Feat/tokens search

### LINKED ISSUE #12826: EVM tokens/ Enhanced searching
search according to token *name*, *symbol* and *token (contract) address* should be possible in chosen category (hidden, collectibles, coins)

Screenshot 2024-06-11 at 17 02 55

### PR DESCRIPTION
**Pull request by AdamSchinzel**
## Description

Make tokens section searchable.

## Related Issue

Resolve #12826

## Screenshots:
Screenshot 2024-06-24 at 10 46 01 AM
Screenshot 2024-06-24 at 10 45 51 AM
Screenshot 2024-06-24 at 10 45 39 AM
Screenshot 2024-06-24 at 10 45 31 AM

### CODE REVIEW

File: packages/suite/src/views/wallet/transactions/TransactionList/TransactionListActions/TransactionListActions.tsx [hunk 1]
```diff
@@ -19,20 +40,89 @@ export const TransactionListActions = ({
...
+                inputRef.current &&
+                (event.ctrlKey || event.metaKey) &&
+                event.key === KEYBOARD_CODE.LETTER_F
+            ) {
+                event.preventDefault();
+                inputRef.current.focus();
+            }
+        },
+        [inputRef],
+    );
```
Comment (tomasklim) [inline packages/suite/src/views/wallet/transactions/TransactionList/TransactionListActions/TransactionListActions.tsx]: please noooo, i hate this on tx history list

File: packages/suite/src/views/wallet/transactions/TransactionList/TransactionListActions/TransactionListActions.tsx [hunk 2]
```diff
@@ -19,20 +40,89 @@ export const TransactionListActions = ({
...
+        document.addEventListener('keydown', onSearchKeys);
+
+        return () => {
+            document.removeEventListener('keydown', onSearchKeys);
+        };
+    }, [account.symbol, account.index, account.accountType, setSearch, onSearchKeys]);
+
+    return (
+        <Wrapper>
+            {isFeatureFlagEnabled('SEARCH_TRANSACTIONS') && (
```
Comment (tomasklim) [inline packages/suite/src/views/wallet/transactions/TransactionList/TransactionListActions/TransactionListActions.tsx]: why?

Comment (AdamSchinzel) [reply packages/suite/src/views/wallet/transactions/TransactionList/TransactionListActions/TransactionListActions.tsx]: It was there before inside the component, was not sure why but didn't want to remove it.

File: packages/suite/src/views/wallet/transactions/TransactionList/TransactionListActions/TransactionListActions.tsx (hunk 1 above)
Comment (AdamSchinzel) [reply packages/suite/src/views/wallet/transactions/TransactionList/TransactionListActions/TransactionListActions.tsx]: So should I remove it from both places?

Comment (tomasklim) [reply packages/suite/src/views/wallet/transactions/TransactionList/TransactionListActions/TransactionListActions.tsx]: I think it makes sense in desktop app but I would remove it from web as it blocks native browser search.

File: packages/suite/src/views/wallet/transactions/TransactionList/TransactionListActions/TransactionListActions.tsx (hunk 2 above)
Comment (tomasklim) [reply packages/suite/src/views/wallet/transactions/TransactionList/TransactionListActions/TransactionListActions.tsx]: let's remove whole `SEARCH_TRANSACTIONS` flag

**Review by tomasklim — APPROVED** [review]

**Pull request merged**